### PR TITLE
Support for native import from Oracle (issue #26)

### PR DESCRIPTION
**Pull request by hendrikcech**
This pull request addresses the feature described by issue #26. Changes include:
- The virtual schema parameters `IMPORT_FROM_ORA` and `ORA_CONNECTION_NAME` are added.
- The function `JdbcAdapter.handlePushdownRequest()` is extended to generate `IMPORT FROM ORA` queries.
- (Almost) all Oracle data types are tested (see class `OracleSqlDialectIT`). All tests are run twice; once over a JDBC and once over an IMPORT FROM ORA connection.
- `oracle-testdata.sql` is included which prepares an Oracle table that is used by the Oracle integration tests.

### CODE REVIEW

File: jdbc-adapter/README.md [hunk 1]
```diff
@@ -88,34 +88,41 @@ DROP VIRTUAL SCHEMA hive CASCADE;
...
 
-Parameter                   | Value
+Property                    | Value
 --------------------------- | -----------
 **SQL_DIALECT**             | Name of the SQL dialect: EXASOL, HIVE, IMPALA, ORACLE, TERADATA, REDSHIFT or GENERIC (case insensitive). If you try generating a virtual schema without specifying this property you will see all available dialects in the error message.
-**CONNECTION_NAME**         | Name of the connection created with ```CREATE CONNECTION``` which contains the jdbc connection string, the username and password. If you defined this property then it is not allowed to set CONNECTION_STRING, USERNAME and PASSWORD. We recommend using this property to ensure that the password will not be shown in the logfiles.
-**CONNECTION_STRING**       | The jdbc connection string. Only required if CONNECTION_NAME is not set.
 
+**Mandatory Connection Specification:**
+Either specify `CONNECTION_NAME` OR provide `CONNECTION_STRING`,String `USERNAME` and `PASSWORD`.
```
Comment (andrehacker) [inline jdbc-adapter/README.md]: A visible new line before Either would be nice to make clear the this is a new subsection.

Comment (andrehacker) [reply jdbc-adapter/README.md]: The "String" word seems to be need needed.

File: jdbc-adapter/README.md [hunk 2]
```diff
@@ -88,34 +88,41 @@ DROP VIRTUAL SCHEMA hive CASCADE;
...
+**SCHEMA_NAME**             | The name of the remote JDBC schema. This is usually case-sensitive, depending on the dialect. It depends on the dialect whether you have to specify this or not. Usually you have to specify it if the data source JDBC driver supports the concepts of schemas.
 **TABLE_FILTER**            | A comma-separated list of table names (case sensitive). Only these tables will be available as virtual tables, other tables are ignored. Use this if you don't want to have all remote tables in your virtual schema.
 
-
 **Advanced Optional Properties:**
 
-Parameter                   | Value
+Property                    | Value
 --------------------------- | -----------
 **IMPORT_FROM_EXA**         | Only relevant if your data source is EXASOL. Either 'TRUE' or 'FALSE' (default). If true, IMPORT FROM EXA will be used for the pushdown instead of IMPORT FROM JDBC. You have to define EXA_CONNECTION_STRING if this property is true.
```
Comment (andrehacker) [inline jdbc-adapter/README.md]: Would be cool if you could change all the SQL statements and property names to monospace, i.e. IMPORT FROM EXA/JDBC and EXA_CONNECTION_STRING here.

Please review at least the Adapter Properties section, I think there are many properties which we could convert to monospace. Then we have this finally consistent. You can make a separate pull request for this documentation fixes.

File: jdbc-adapter/README.md [hunk 3]
```diff
@@ -137,9 +144,11 @@ You have to make sure that Exasol can connect to the host running the udf_debug.
...
 See https://docs.oracle.com/javase/7/docs/api/java/sql/DriverManager.html
-
+* **Very slow execution of queries with SCRIPT_OUTPUT_ADDRESS**: If `SCRIPT_OUTPUT_ADDRESS` is set as explained in the [debugging section](#debugging), verify that a service is actually listening at that address. Otherwise, if Exasol can not establish a connection, repeated connection attempts can be the cause for slowdowns.
+* **Very slow execution of queries**: Depending on which JDK version Exasol uses to execute Java user-defined functions, a blocking randomness source may be used by default. Especially cryptographic operations do not complete until the operating system has collected a sufficient amount of entropy. This problem seems to occur most often when Exasol is run in an isolated environment, e.g., a virtual machine or a container. A solution is to use a non-blocking randomness source. 
+  To do so, log in to EXAOperation and shutdown the database. Append `-etlJdbcJavaEnv -Djava.security.egd=/dev/./urandom` to the "Extra Database Parameters" input field and power the database on again.
```
Comment (andrehacker) [inline jdbc-adapter/README.md]: This will fix the slowness of the IMPORT FROM JDBC, however, there are also JDBC metadata calls directly from the Adapter script, which will be executed in a nested container environment.
If this is the case, the %env (or javaoptions, forgot the details) would need to be set in the adapter script.
Did you also do this to fix the slowness issues?

Comment (hendrikcech) [reply jdbc-adapter/README.md]: I'm not so sure about this case. Before submitting this PR I did some quick tests with setting the Java options in both EXAOperations and the adapter script (using `%env` variables), setting them in only one of the two, and not setting them at all. Here are the results running the integration tests two times for each combination (`y` meaning "Java option set", `n` meaning "option not set"):
```
EXAOperation y, Adapter Script n: 06:32 min / 6:32 min
EXAOperation y, Adapter Script y: 06:30 min / 6:46 min
EXAOperation n, Adapter Script y: 29:47 min / 31:09 min
EXAOperation n, Adapter Script n: 14:35 min / 14:50 min
```
The results are pretty surprising (especially comparing the last two cases) but that's what I based my recommendation on.
